packages/field-base/src/field-mixin.js

```diff
@@ -145,7 +145,7 @@ export const FieldMixin = (superclass) =>
     __labelChanged(hasLabel, labelNode) {
       // Label ID should be only added when the label content is present.
       // Otherwise, it may conflict with an `aria-label` attribute possibly added by the user.
-      if (hasLabel) {
+      if (hasLabel && !this.accessibleName && !this.accessibleNameRef) {
```

Comment:
Should the whole observer be skipped if any of `accessibleName` or `accessibleNameRef` are provided?

Reply:
Makes sense. Refactored.